src/blockmetadata.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+Minetest
+Copyright (C) 2010-2013 celeron55, Perttu Ahola <celeron55@gmail.com>
```

Comment:
By now it would be safe to just mention yourself and the most active metadata (overall) contributor in this header.

Reply:
The code was mostly copied from `nodemetadata.cpp`.